<launch>
               
		<node name="flight_controller" pkg="tiltrotor_drone" type="flight_control_exe" output="screen">
		<param name="node_rate" value="50.0"/>
		<param name="drone_name" value="iris_tiltrotor"/>
		<param name="kp_tau_x" value="0.4"/>
		<param name="ki_tau_x" value="0.0"/>
		<param name="kd_tau_x" value="0"/>
		<param name="kp_tau_y" value="0.4"/>
		<param name="ki_tau_y" value="0.0"/>
		<param name="kd_tau_y" value="0"/>
		<param name="kp_tau_z" value="0.4"/>
		<param name="ki_tau_z" value="0.0"/>
		<param name="kd_tau_z" value="0"/>
		<param name="m" value="5.2"/>
		<param name="I_x" value="0.026"/>
		<param name="I_y" value="0.15"/>
		<param name="I_z" value="0.146"/>
	</node>
</launch>
